charts/ditto/templates/nginx-configmap.yaml

```diff
@@ -9,6 +9,7 @@
 #
 # SPDX-License-Identifier: EPL-2.0
 ---
+{{- if .Values.nginx.enabled -}}
```

Comment:
I am not sure if the `-` in `-}}` is correct. FMPOV it will lead to `apiVersion: v1` being pulled up directly behind the `---` delimiter ...
@BobClaerhout can you check by running `helm template` locally?

Reply:
I do not get any errors when running `helm template` locally. 
Also, this is a copy from https://github.com/eclipse/packages/blob/a21fb8306e1bc2116efeda6c02cb023ba2960ec5/charts/ditto/templates/nginx-config.yaml#L12 so this should be ok.